pkgs/development/libraries/glibmm/2.68.nix

```diff
@@ -13,13 +13,13 @@
 
 stdenv.mkDerivation rec {
   pname = "glibmm";
-  version = "2.76.0";
```

Comment:
Filename indicates 2.68, making a PR with 2.78

Reply:
2.68 is the API version, which is not changed in 2.78
https://gitlab.gnome.org/GNOME/glibmm/-/blob/2.78.0/meson.build?ref_type=tags#L14

Reply:
oh :o dropping that commit and this comment.